presto-hive/src/main/java/com/facebook/presto/hive/HiveUtil.java

```diff
@@ -332,7 +332,7 @@ private static Deserializer createDeserializer(Class<? extends Deserializer> cla
     private static void initializeDeserializer(Deserializer deserializer, Properties schema)
     {
         try {
-            deserializer.initialize(null, schema);
+            deserializer.initialize(new Configuration(), schema);
```

Comment:
This is reasonable. The only downside over null is that if there is a deserializer that uses it and expects non-default configuration, it will be harder to track down the failure reason (NPE is easy).